### ISSUE TITLE
Tooltip on hover for faceted charts

### ISSUE THREAD

**Issue by larsyencken** [owid/owid-grapher#1084]
## Overview

When you mouse-over a normal chart, you get a small infobox giving you a summary of the thing you're hovering on. For example, for line charts, you get the values of all series at the time point you're hovering on. But faceted charts are missing this interactivity, making it much harder to read off an exact value from a faceted chart.

## Context

When implementing faceted charts, we did not have the opportunity to complete this part. But it's the most glaring missing piece right now.

**Comment by marcopiani** [comment]
Hi!

I see this is now implemented! Great job!

One idea/request: would it be possible to have the option (to be selected with a checkbox, perhaps) to have the tooltip shown in parallel on all subplots?

See example at [https://projects.fivethirtyeight.com/biden-approval-rating/](https://projects.fivethirtyeight.com/biden-approval-rating/)
![image](https://user-images.githubusercontent.com/18381804/169087204-85130d7f-9548-4406-9c1b-6badb498c6dd.png)

**Comment by danyx23** [comment]
@marcopiani thanks for your continued interest! Yes we are aware that this would be nice to have but in our case we have to find enough screen real estate to show all those tooltips and make sure they don't overlap each other on smaller screens etc. We will likely continue work on this but we thought it is important enough to ship this first step and tackle this other improvement later.

**Comment by marcopiani** [comment]
Hi @danyx23 many thanks for the prompt reply.

I understand the need to give priority to other features.

Thank you for considering the option, and, as always, for the work of the whole OWID team.
